integreat_cms/api/v3/regions.py

```diff
@@ -34,7 +34,8 @@ def transform_region(region):
         "latitude": region.latitude,
         "bounding_box": region.bounding_box.api_representation,
         "aliases": region.aliases,
-        "tunews": region.tunews_enabled,
+        "tunews": region.external_news_enabled,
```

Comment:
> I think it would make sense to rename the database field as well, and only keep the old name in the API for legacy reasons.

I changed the database name now and kept this entry in the API. But I switched it to the new field, since the old field was removed. Is that correct or should I revert the migration partly to restore the tunews_enabled, so nothing is changed?

Reply:
Ah, and one thing I forgot: Since we have [API documentation](https://github.com/digitalfabrik/integreat-cms/blob/develop/docs/src/api-docs.rst) now, could you document the new field?